src/main.js

```diff
@@ -314,6 +309,8 @@ const saveMetaDataSingleFile = (_editionCount) => {
         `Writing metadata for ${_editionCount}: ${JSON.stringify(metadata)}`
       )
     : null;
+  
+  delete metadata.edition;
```

Comment:
Why do you delete it, when you can just delete the line what includes this data?

What's the point of `add edition` and then `delete edition`?

Reply:
The edition is used on saveMetaDataSingleFile as well, as a reference to get the right set of data. 
You could edit saveMetaDataSingleFile and use the name of the image for instance, or change the structure of it but mine was just a quick fix. 

One thing that I got wrong is that it's necessary an if statement to check if the network is solana.
See review below.